commands-yml/commands/element/actions/send-keys.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: Send Keys
+short_description: Send a sequence of key strokes to an element
+description:
+  >
+    Any UTF-8 character may be specified, however, if the server does not support native key events, it should simulate key strokes for a standard US keyboard layout. The Unicode Private Use Area code points, 0xE000-0xF8FF, are used to represent pressable, non-text keys (see table below).
+
+example_usage:
+  java:
+    |
+      MobileElement element = (MobileElement) driver.findElementByAccessibilityId("SomeAccessibilityID");
+      element.sendKeys("Hello world!");
+  python:
+    |
+      self.driver.find_element_by_accessibility_id('SomeAccessibilityID').send_keys('Hello world!')
+  javascript_wd:
+    |
+      let element = await driver.elementByAccessibilityId("SomeAccessibilityID");
+      await element.type("Hello world!")
+  javascript_wdio:
+    |
+      driver.setValue("~SomeAccessibilityId");
+  ruby:
+    |
+      @driver.find_element(:accessibility_id, "SomeAccessibilityID").send_keys("Hello World!"")
```

Comment:
I think there's an extra `"` in `"Hello World!""`

Reply:
Good eye. Thanks!